data/json/mapgen/helipad.json

```diff
@@ -0,0 +1,215 @@
+        ".F..=,,,,,,,==o===,,,,,,,,,,,,===o=,,,,,,,==..F.",
+        ".F..=,,,,,,,==o===,,========,,===o=,,,,,,,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..=,=====,==o===,,========,,===o=,=====,==..F.",
+        ".F..==========o==================o==========..F.",
+        ".F..==========o==================o==========..F.",
+        "FFG............FGF............FGF............GFF",
+        "FTFFFFFFFFFFFFFFTFFFFFFFFFFFFFFTFFFFFFFFFFFFFFTF",
+        "FFF............FFF............FFF............FFF"
+      ],
+      "palettes": [ "helipad_palette" ],
+      "toilets": { "&": {  } },
+      "gaspumps": { "$": {  } },
+      "items": {
+        "B": {
+          "item": {
+            "subtype": "distribution",
+            "entries": [
+              { "item": "gasoline", "charges": [ 0, 80 ], "container-item": "30gal_drum", "prob": 25 },
+              { "item": "motor_oil", "charges": [ 0, 30 ], "container-item": "jerrycan", "prob": 5 },
+              { "item": "30gal_drum", "prob": 70 }
+            ]
+          },
+          "chance": 75
+        },
+        "C": {
+          "item": {
+            "subtype": "distribution",
+            "entries": [
+              { "group": "military_patrol_food", "prob": 50 },
+              { "group": "infantry_common_gear", "prob": 5 },
+              { "group": "mil_food", "prob": 40 },
+              { "group": "infantry_medical_gear", "prob": 5 },
+              { "group": "supplies_metal", "prob": 150 },
+              { "group": "supplies_mechanics", "prob": 50 }
+            ]
+          },
+          "chance": 75,
+          "repeat": [ 3, 5 ]
+        },
+        "r": {
+          "item": {
+            "subtype": "distribution",
+            "entries": [ { "group": "military_standard_assault_rifles", "prob": 45 }, { "group": "military_standard_shotguns", "prob": 5 } ]
+          },
+          "chance": 10
+        },
+        "l": {
+          "item": {
+            "subtype": "distribution",
+            "entries": [
+              { "group": "army_personal_locker", "prob": 50 },
+              { "group": "military_patrol_food", "prob": 25 },
+              { "group": "supplies_plumbing", "prob": 50 }
+            ]
+          },
+          "chance": 33,
+          "repeat": [ 1, 5 ]
+        },
+        "L": { "item": "army_personal_locker", "repeat": [ 0, 4 ] },
+        "d": { "item": "SUS_office_desk" },
+        "f": { "item": "SUS_office_filing_cabinet" },
+        "4": { "item": "military_standard_shotguns", "chance": 50 },
+        "#": { "item": "SUS_utensils", "chance": 25, "repeat": [ 1, 3 ] },
+        "t": { "item": "SUS_dishes", "chance": 50, "repeat": [ 1, 2 ] },
+        "[": { "item": "SUS_fridge" },
+        "O": { "item": "SUS_oven" },
+        "H": { "item": "SUS_spice_collection" }
+      },
+      "place_monsters": [
```

Comment:
I don't know the groups well but I'd assume the regular, non-weak military groups should also have a chance of spawning here if it's a major outpost with a helipad.

Reply:
Tweaked that a bit. 
There are three military monstergroups, GROUP_MIL_WEAK, GROUP_MIL_STRONG and GROUP_MAYBE_MIL. First two have different sizes and stronger Zs in latter case, third one also adds military robots to the mix.